README.md

```diff
@@ -117,6 +117,8 @@ Suspenders also comes with:
 
 ## Heroku
 
+Read the documentation on [deploying to Heroku](heroku deploy)
```

Comment:
Broken Link:

```suggestion
Read the documentation on [deploying to Heroku][heroku deploy]
```

Reply:
Now I am getting a 404 for the link

Reply:
Hey, apologies. I think there are two issues:

1. The link was actually wrong (didn't contain `/blob/master/`
2. The link won't work until this is actually merged to `master`

Reply:
Thanks!